packages/server/src/db/index.js

```diff
@@ -198,6 +198,7 @@ async function getGrants({
     const { data, pagination } = await knex(TABLES.grants)
         .select(`${TABLES.grants}.*`)
         .modify((queryBuilder) => {
+            queryBuilder.distinctOn(`${TABLES.grants}.grant_id`);
```

Comment:
I looked up `DISTINCT ON` because I'd never seen it before. I learned that it is Postgres specific, but I don't know if that is a concern. My limited understanding is from [here](https://www.geekytidbits.com/postgres-distinct-on/), which says the `ORDER BY` clause plays a role in processing `DISTINCT ON`, but the code seems to allow for a query with no ordering. Again, not sure if that's a concern.

Reply:
I was thinking of adding a default sort by on created column but didnt. Ill test it out and add it